### PR TITLE
go oasis-node: close stdout before test framework prints junk

### PR DESCRIPTION
**Pull request by pro-wh**
blocks #2344

there's no option to disable messages from the test framework

### CODE REVIEW

File: go/oasis-node/integrationrunner/integration_test.go [hunk 1]
```diff
@@ -16,4 +17,7 @@ func TestIntegration(t *testing.T) {
 	}
 
 	launch()
+
+	// Suppress coverage report on stdout.
+	_ = os.Stdout.Close()
```
Comment (Yawning) [inline go/oasis-node/integrationrunner/integration_test.go]: Do we only care about stdout?

Also, `os.Stdout = ioutil.Discard` might be slightly better behaved, though file descriptor re-use is unlikely.

Comment (pro-wh) [reply go/oasis-node/integrationrunner/integration_test.go]: I wanted that too, but the types don't match.

Comment (pro-wh) [reply go/oasis-node/integrationrunner/integration_test.go]: and yeah the issue that this fixes is only about stdout.

Comment (Yawning) [reply go/oasis-node/integrationrunner/integration_test.go]: Oh, it's an `os.File`, so something like:
```
  f, err := os.OpenFile(os.DevNull, os.O_RW, 0)
  if err != nil {
    t.Fatalf("Failed to open /dev/null") // Use testify if you want, whatever.
  }
  os.Stdout = f
```

Comment (pro-wh) [reply go/oasis-node/integrationrunner/integration_test.go]: ok

**Pull request merged**